src/k8s/pkg/k8sd/app/hooks_remove.go

```diff
@@ -66,9 +66,11 @@ func (a *App) onPreRemove(ctx context.Context, s state.State, force bool) (rerr
 				log.Error(err, "Failed to create Kubernetes client", err)
 			}
 
-			log.Info("Deleting node from Kubernetes cluster")
-			if err := c.DeleteNode(ctx, s.Name()); err != nil {
-				log.Error(err, "Failed to remove Kubernetes node")
+			if c != nil {
```

Comment:
@mateoflorido drive-by fix of KU-1270 as I was running into this issue while debugging